lib/reek/ast/sexp_extensions/constant_assignment.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+module Reek
+  module AST
+    module SexpExtensions
+      # Utility methods for :casgn nodes.
+      module CasgnNode
+        def assignment
```

Comment:
I find this name confusing. Assignment to me means an expression of the form `a = b`. The thing returned from this method would be just `b`. Perhaps `value` would be a better name for this method.